apps/bot/src/lib/starboard/webhook.ts

```diff
@@ -31,7 +31,14 @@ export async function createWebhookMessage(
 		.setStyle(ButtonStyle.Link)
 		.setURL(message.url)
 		.setEmoji({ name: "🔗" })
-		.setLabel("View Original");
+		.setLabel(
+			`Posted in #${channel.name.length > 69 ? channel.name.slice(0, 66) + "..." : channel.name}`
+		);
+
+ 	const attachments = [...message.attachments.values()];
+				? `Posted in #${channel.name}`.slice(0, 77) + "…"
+				: `Posted in #${channel.name}`
+		);
```

Comment:
lol nice merge Ian top tier skill

```suggestion
		.setLabel(
			`Posted in #${channel.name.length > 68 ? channel.name.slice(0, 67) + "…" : channel.name}`
		);

```